common/transport/http/src/http.ts

```diff
@@ -19,6 +19,23 @@ export type HttpMethod = 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE';
  */
 export type HttpCallback = (err: Error, body?: string, response?: IncomingMessage) => void;
 
+/**
+ * This interface defines optional HTTP request options that one can set on a per request basis.
+ */
```

Comment:
we need @private tag on those to make sure no doc gets generated (although the whole package should be excluded... better to have belts and suspenders)

Reply:
Sure.